sql/plan/scope.go

```diff
@@ -92,9 +93,13 @@ func (s *Scope) NewScope(node sql.Node) *Scope {
 
 // NewScopeFromSubqueryExpression returns a new subscope created from a subquery expression contained by the specified
 // node.
-func (s *Scope) NewScopeFromSubqueryExpression(node sql.Node) *Scope {
+func (s *Scope) NewScopeFromSubqueryExpression(node sql.Node, corr sql.ColSet) *Scope {
```

Comment:
Would be helpful to update the method doc to mention the `corr` parameter and explain what it means here. It's probably painfully obvious to you of course, but a little documentation there would help other people from having to reverse engineer it when they see this code for the first time.

Reply:
added some attribute docs for what it means, and doc comment here to indicate why it needs to be explicit for subquery expressions